src/worker.cc

```diff
@@ -415,6 +415,11 @@ void parse_locations(const rapidjson::Document& doc,
         }
       } catch (...) { throw valhalla_exception_t{location_parse_error_code}; }
     }
+    // first and last locations get the default type of break no matter what
+    if (locations->size()) {
+      locations->Mutable(0)->set_type(odin::Location::kBreak);
```

Comment:
i was thinking we should enforce that

Reply:
we do it down stream in thor but we should have done it upstream here..